api/src/test/java/io/opencensus/trace/EndSpanOptionsTest.java

```diff
@@ -26,19 +26,20 @@
 /** Unit tests for {@link EndSpanOptions}. */
 @RunWith(JUnit4.class)
 public class EndSpanOptionsTest {
-  @Test(expected = NullPointerException.class)
-  public void setNullStatus() {
-    EndSpanOptions.builder().setStatus(null).build();
```

Comment:
It might be useful to test clearing the status by setting the status and then setting it back to null.

Reply:
EndSpanOptions.Builder#setStatus does not accept null Status anymore. No reason for the builder to support set/unset/set in my opinion.